backend/copr_backend/actions.py

```diff
@@ -56,18 +56,18 @@ def create_from(cls, opts, action, log=None):
     def get_action_class(cls, action):
         action_type = action["action_type"]
         action_class = {
-            ActionType.LEGAL_FLAG: LegalFlag,
-            ActionType.CREATEREPO: Createrepo,
-            ActionType.UPDATE_COMPS: CompsUpdate,
-            ActionType.GEN_GPG_KEY: GenerateGpgKey,
-            ActionType.RAWHIDE_TO_RELEASE: RawhideToRelease,
-            ActionType.FORK: Fork,
-            ActionType.BUILD_MODULE: BuildModule,
-            ActionType.DELETE: Delete,
-            ActionType.REMOVE_DIRS: RemoveDirs,
+            ActionTypeEnum("legal-flag"): LegalFlag,
```

Comment:
Probably a typo, the rest of the action names use underscores instead.

Reply:
Seems like a typo that I didn't want to change, that would be a common-API change, not sure if worth it.